.travis.yml

```diff
@@ -8,6 +8,7 @@ node_js:
   - 8
 
 script:
+  - npm run compile && npm run component
```

Comment:
Could you please split this into two lines?

Reply:
All done.